Slambook/index.js

```diff
@@ -34,6 +34,7 @@ form.addEventListener("submit", (e) => {
     validateInput(beautifulMessage);
     validateInput(nickname);
     validateInput(songForMe);
+    validateInput(shareOpinion);
     

```

Comment:
allSlamArray is not defined before being used, make sure to define it before using it in the forEach loop.